addons/medical/functions/fnc_isMedic.sqf

```diff
@@ -19,7 +19,7 @@
 
 params ["_unit", ["_medicN", 1]];
 
-private _class = _unit getVariable [QGVAR(medicClass), getNumber (configFile >> "CfgVehicles" >> typeOf _unit >> "attendant")];
+private _class = _unit getVariable [QGVAR(medicClass), _unit getUnitTrait "medic"];
```

Comment:
It might be better to introduce an extra config property to support doctors that way (although, are doctors even relevant in the current medical code?)